extensions/amp-audio/amp-audio.md

```diff
@@ -131,6 +131,40 @@ Example:
 </amp-audio>
 ```
 
+## Analytics
+AMP audio analytics gathers data about how users interact with audios in AMP documents. AMP audio extension issues various analytics events during their lifecycle. These events can be reported through the analytics configuration using `audio-*` triggers.
```

Comment:
nit: Let's add that `play` and `pause` are the two analytics events we support now.